docs-ref-conceptual/install-azure-cli-apt.md

```diff
@@ -24,15 +24,13 @@ for the Azure CLI. This package has been tested with:
 1. <a name="install-step-1"/> Modify your sources list:
 
     ```bash
-    AZ_REPO=$(lsb_release -cs)
-    echo "deb [arch=amd64] https://packages.microsoft.com/repos/azure-cli/ $AZ_REPO main" | \
-        sudo tee /etc/apt/sources.list.d/azure-cli.list
+     sudo apt-add-repository "deb [arch=amd64] https://packages.microsoft.com/repos/azure-cli/ $(lsb_release -cs) main"
```

Comment:
Information on what to do if `apt-add-repository` is missing needs to be added to troubleshooting. It is part of the `software-properties-common` package which may not be installed.